app/components/chart-donut.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
       return;
     }
 
+    if (!isIcon) {
+      svg.classed("svg-container", true)
```

Comment:
I will give it a try and let you know!  But I think the currently solution we have is the simplest one. I will think this through and get back to you.